spec/acceptance/cs_commit_spec.rb

```diff
@@ -72,9 +72,7 @@ class { 'corosync':
     EOS
 
     apply_manifest(pp, debug: true, trace: true, catch_failures: true)
-    # Another run is needed to update the Puppet shadow CIB
-    apply_manifest(pp, expect_changes: true)
-    # But the last one should not change anything
+    # Test for indempotency
     apply_manifest(pp, catch_changes: true)
```

Comment:
Isn't that what `catch_changes` does?  Previously it was run 3 times.  Only the 3rd run had to check for no changes.